k8s/k6/k6-deployment.yaml

```
@@ -0,0 +1,37 @@
+              value: "true"
+          resources:
+            requests:
+              cpu: "100m"
+              memory: "128Mi"
+            limits:
+              cpu: "500m"
+              memory: "512Mi"
+      volumes:
+        - name: k6-scripts
+          configMap:
+            name: k6-scripts
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)